### PR TITLE
Add run-migrate in Makefile

### PR DESCRIPTION
**Pull request by ThiagoLopes**
Only one implementation

### CODE REVIEW

File: inspace/core/urls.py [hunk 1]
```diff
@@ -4,6 +4,9 @@
 
 
 urlpatterns = [
-    url(r'^resources/new/$', views.resource_create_view, name='resource-create'),
+    url(r'^resources/new/$',
+        views.resource_create_view, name='resource-create'),
+    url(r'^resources/search/(?P<title>\w{0,50})$',
```
Comment (cacarrara) [inline inspace/core/urls.py]: * We do not need `search` in this url.
* I would suggest to use here the default GET url format. `/resources?title=searched_resource_title`. The format `/resources/<something>` should return a specific resource (or 404 if not found) and not a list of resources.

File: inspace/frontend/templates/core/resource_list.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```
Comment (cacarrara) [inline inspace/frontend/templates/core/resource_list.html]: We do not need html here. I would suggest to use the ready sample template `results.html` in frontend folder.

File: inspace/tests/core/test_views.py [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
 def test_home_get_successfully(client):
     response = client.get(reverse('core:home'))
     assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_resource_list_successfully(client):
+    response = client.get(
+        reverse('core:resource-list', kwargs={'title': 'python'}))
+    assert response.status_code == 200
```
Comment (cacarrara) [inline inspace/tests/core/test_views.py]: We can put more tests here. For example: givin an fixture with N resources, the list endpoint must return a list with all resources unfiltered. Other test: given an fixture with N resources, the list must return a list with L resources that match with provided title.

File: inspace/frontend/templates/core/resource_list.html (hunk 2 above)
Comment (ThiagoLopes) [inline inspace/frontend/templates/core/resource_list.html]: Should not have gone to the PR. Sorry

File: inspace/core/urls.py (hunk 1 above)
Comment (ThiagoLopes) [inline inspace/core/urls.py]: Ok!

File: inspace/tests/core/test_views.py (hunk 3 above)
Comment (ThiagoLopes) [inline inspace/tests/core/test_views.py]: It will be done

**Pull request closed without merging**